docs/testrunner-toolkit/running-tests.md

```diff
@@ -28,14 +28,14 @@ Unless you specify a test directory, `saucectl` executes tests based on the test
 
 The following steps outline how to run your tests on your local machine with the containerized solution. This is the preferred option if you wish to run tests on your local machine, or if you wish to accelerate test execution in CI.
 
-Please note that if you don't specify a `--test-env` flag when running tests, `saucectl` defaults to `sauce` mode.
+Please note that if you don't specify a `--mode` flag when running tests, `saucectl` defaults to `sauce` mode.
```

Comment:
I think this is no longer true. If you do not specify a runtime mode flag, the test will default to 1) the default config; or 2) the suite-specific config. It will ONLY default to `sauce` is none of those values is set.